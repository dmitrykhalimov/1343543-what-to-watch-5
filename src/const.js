export const ratingRanks = [
  {
    'Bad': 2.9
  },
  {
    'Normal': 4.9
  },
  {
    'Good': 7.9
  },
  {
    'Very good': 9.9
  },
  {
    'Awesome': 10
  }
];
