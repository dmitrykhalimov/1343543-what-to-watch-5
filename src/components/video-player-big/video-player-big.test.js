import React from "react";
import renderer from "react-test-renderer";
import VideoPlayerBig from "./video-player-big";
import {MemoryRouter} from 'react-router-dom';
import {TEST_MOCKS} from "../../const";

describe(`VideoPlayerBig`, () => {
  it(`Should VideoPlayerBig render correctly`, () => {
    const tree = renderer
      .create(
          <MemoryRouter>
            <VideoPlayerBig
              id={`1`}
              isPlaying={true}
              film={TEST_MOCKS.film}
              onPlayPauseClick={TEST_MOCKS.noop}
              onFullscreenClick={TEST_MOCKS.noop}
            />
          </MemoryRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
