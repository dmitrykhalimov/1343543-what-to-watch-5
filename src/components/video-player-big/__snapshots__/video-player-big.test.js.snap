// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayerBig Should VideoPlayerBig render correctly, isPlaying 1`] = `
Array [
  <video
    className="player__video"
    poster="/img/player-poster.jpg"
    src="http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4"
  />,
  <a
    className="player__exit"
    href="/films/29"
    onClick={[Function]}
    type="button"
  >
    Exit
  </a>,
  <div
    className="player__controls"
  >
    <div
      className="player__controls-row"
    >
      <div
        className="player__time"
      >
        <progress
          className="player__progress"
          max="100"
          value="0"
        />
        <div
          className="player__toggler"
          style={
            Object {
              "left": "0%",
            }
          }
        >
          Toggler
        </div>
      </div>
      <div
        className="player__time-value"
      >
        1:30:29
      </div>
    </div>
    <div
      className="player__controls-row"
    >
      <button
        className="player__play"
        onClick={[Function]}
        type="button"
      >
        <svg
          height="19"
          viewBox="0 0 19 19"
          width="19"
        >
          <use
            xlinkHref="#play-s"
          />
        </svg>
        <span>
          Play
        </span>
      </button>
      <div
        className="player__name"
      >
        Interstate 60: Episodes of the Road
      </div>
      <button
        className="player__full-screen"
        onClick={[Function]}
        type="button"
      >
        <svg
          height="27"
          viewBox="0 0 27 27"
          width="27"
        >
          <use
            xlinkHref="#full-screen"
          />
        </svg>
        <span>
          Full screen
        </span>
      </button>
    </div>
  </div>,
]
`;
